### PR TITLE
[TH2-2533] Add attached messages selection to messages panel

### PR DESCRIPTION
**Pull request by osinkinroman**

### CODE REVIEW

**Comment by osinkinroman** [comment]
Layout from https://www.figma.com/file/vB9EyLv1GBE3vJqy2Hxk08/TH2-New-Report-Style-Updates?node-id=2498%3A66935

**Comment by osinkinroman** [comment]
In Figma its in TimeLine and Serch page

File: src/components/message/MessagesAttachedSelection.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```
Comment (rzvdv) [inline src/components/message/MessagesAttachedSelection.tsx:1]: add license

File: src/components/message/MessagesAttachedSelection.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+const MessageAttachedSelection = () => {
+	const messagesStore = useMessagesWorkspaceStore();
+
+	const { attachedMessages } = messagesStore;
+
+	const [currentAttachedMessages, setCurrentAttachedMessages] = React.useState<EventMessage[]>([]);
+	const [messageIndex, setMessageIndex] = React.useState<number>(0);
+
+	React.useEffect(() => {
+		if (attachedMessages !== currentAttachedMessages) {
```
Comment (rzvdv) [inline src/components/message/MessagesAttachedSelection.tsx:15]: i think we dont even need to compare it, hook will be called every time we load new attached messages so we can delete state `currentAttachedMessages`

**Review by rzvdv — CHANGES_REQUESTED** [review]

File: src/components/message/MessagesAttachedSelection.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	};
+
+	if (attachedMessages.length === 0) return null;
+
+	return (
+		<div className='messages-list__attached-messages'>
+			<button className='messages-list__attached-messages-btn' onClick={onPrevious}>
+				<div className='messages-list__attached-messages-btn-previous' />
+			</button>
+			<span className='messages-list__attached-messages-text'>Show previous</span>
```
Comment (rzvdv) [inline src/components/message/MessagesAttachedSelection.tsx:53]: make text to be part of the button

File: src/components/message/MessagesAttachedSelection.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************** */
+
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { useMessagesWorkspaceStore } from '../../hooks';
+
+const MessageAttachedSelection = () => {
```
Comment (rzvdv) [inline src/components/message/MessagesAttachedSelection.tsx:21]: i think it should be AttachedMessageSelection

**Review by rzvdv — APPROVED** [review]

**Pull request merged**